scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -65,9 +81,9 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                             <span class="stage-group__desk">{{ monitoring.singleGroup.header }}</span>
                             <span class="stage-group__split">/</span>
                             <div class="dropdown" dropdown>
-                                <button class="dropdown__toggle navbtn navbtn--text-only navbtn--padding-s" dropdown-toggle>{{ monitoring.singleGroup.subheader}} {{ monitoring.singleGroup.type | splitText | translate }}<span class="dropdown__caret"></span></button>
+                                <button class="dropdown__toggle navbtn navbtn--text-only navbtn--padding-s" dropdown-toggle>{{ monitoring.singleGroup.subheader}} {{ gettext(monitoring.singleGroup.type) | splitText }}<span class="dropdown__caret"></span></button>
```

Comment:
Don't use `gettext` on a dynamic value. It should come here already translated. Translate it where it's being set to a static string.